.gitignore

```diff
@@ -55,3 +55,4 @@ docs/_build/
 
 # PyBuilder
 target/
+.DS_Store

```

Comment:
You're ignoring this file, but also you added it in your initial commit!  You should remove that add from the commit.  (In general, don't do git add -A).

Reply:
Yes, sorry, I got a bit confused about this. I'm using github desktop at the moment, which seems to alway do git add -A unless I specifically remove files I don't want to commit.